### ISSUE TITLE
Speaker information given as attendee in ICal files aren't picked up

### ISSUE THREAD

**Issue by snoack** [Wilm0r/giggity#69]
If you look at events on the FOSDEM schedule (provided from the built-in list of conferences) for example, speaker names are given at the top of the events. So the UI is obviously able to show speaker information.

However, if you import an iCal file by it's URL it doesn't show given attendees as speakers. To reproduce use FOSDEM's iCal file for example: https://fosdem.org/2018/schedule/ical

**Comment by mmuman** [comment]
Yes, that's because the iCal format is abused as a conference description format, but not everyone agree on the way to insert speaker info inside. Some put it in the title inside brackets or after a dash, some add them as attendees of the meeting.
Currently Giggity doesn't parse this.
I started adding support but it's not finished and not integrated yet, see #28 .
